* Fix issue with `context.auth.uid` in callable Cloud Functions (#2057).